src/solver/residuals.F90

```diff
@@ -28,7 +28,18 @@ subroutine residual_block
     integer(kind=intType) :: i, j, k, l
     integer(kind=intType) :: iale, jale, kale, lale, male ! For loops of ALE
     real(kind=realType), parameter :: K1 = 1.05_realType
-    real(kind=realType), parameter :: K2 = 0.6_realType ! Random given number
+    ! K2
```

Comment:
the changes below were a trial to make the original lowspeed preconditioner work. you can keep these lines as an explanation, but the preconditioner did not work in the end, so mads did not use this. the modified code is just used for the low-speed preconditioner part of this routine. so maybe you can add comments explaining this a bit?

Reply:
I commented with your words, let me know if more description is needed